lib/schemas.js

```diff
@@ -94,17 +97,27 @@ function normalizeSchema (routeSchemas) {
 }
 
 function generateFluentSchema (schema) {
-  for (const key of SCHEMAS_SOURCE) {
-    if (schema[key] && (schema[key].isFluentSchema || schema[key][kFluentSchema])) {
-      schema[key] = schema[key].valueOf()
+  // for (const key of SCHEMAS_SOURCE) {
+  //   if (schema[key] && (schema[key].isFluentSchema || schema[key][kFluentSchema])) {
+  //     schema[key] = schema[key].valueOf()
+  //   }
+  // }
```

Comment:
why is this commented out?

Reply:
Probably leftovers from me editing that I forgot to remove